app/code/community/TIG/MyParcel2014/Model/Api/MyParcel.php

```diff
@@ -274,6 +274,33 @@ public function getRequestErrorDetail()
         return false;
     }
 
+	/**
+	 *
+	 * Get the Megento version and MyParcel version
+	 *
+	 * @return $this
+	 */
+	protected function getUserAgent()
```

Comment:
Ik zou de gehele functie plaatsen onder _setRequestParameters. Aangezien het Magento 1 is, moet je private functies prefixen met een _